types/dropzone/index.d.ts

```diff
@@ -169,9 +181,13 @@ declare class Dropzone {
     static ERROR: string;
     static SUCCESS: string;
 
+    element: string | HTMLElement;
```

Comment:
It won't be `string` in the end
See
https://gitlab.com/meno/dropzone/-/blob/6cef0c303a472468f8d3581c82062a6d31045666/src/dropzone.js#L936

Reply:
Good catch, thank you!